### DISCUSSION TITLE
Albums are not shown

### DISCUSSION THREAD

**Discussion by achatzi** [rern/rAudio#191] [Fixed Issues]
I have just installed the latest rune os
rAudio 1 • 202102121
5.10.13-1-ARCH armv7l
Raspberry Pi 4 Model B Rev 1.4
BCM2711 • 7.6GiB
4 Cortex-A72 @ 1.5GHz

and I have created the db (from my 2tb usb hdd).
It shows that it has 1519 artists, 408 genres etc but no Albums.

What I'm doing wrong? How can I check the logs to see why the system hasn't recognised any album?
The metadata are correct, the same hdd is working fine with both volumio and moode.

**Comment by rern** [comment]
Try:
- Command: `mpc -f '%album%^^[%albumartist%|%artist%]^^%file%' listall`
- Post errors if any.
- Copy an album to `/mnt/MPD/SD`.
- Remove USB drive.
- Update Library database (The icon next to Settings >Library)

**Reply by achatzi** [reply]
rAudio:~  #  mpc -f '%album%^^[%albumartist%|%artist%]^^%file%' listall
MPD error: Connection closed by the server

**Reply by rern** [reply]
See link: [Albums not show in view "Album"](https://github.com/rern/rAudio-1/discussions/71)
Set output buffer to fix it: Settings > MPD > Options - Output Buffer

**Reply by achatzi** [reply]
Increasing the Options - Output Buffer worked fine. Thanks for your help.
